pyccel/parser/syntactic.py

```diff
@@ -172,10 +176,10 @@ def __init__(self, inputs, **kwargs):
                           severity='fatal')
 
         preprocess_imports(red)
-        preprocess_default_args(red)
```

Comment:
@saidctb Can you explain why we don't need to preprocess the default arguments anymore?

Reply:
because it's handled in FunctionCall where we add the missing argument if they where optional